.hadolint.yaml

```diff
@@ -0,0 +1,2 @@
+ignored:
+  - DL3008
```

Comment:
What is this file used for?

Reply:
This is `hadolint` configuration file, see [hadolint configuration  ]( https://github.com/hadolint/hadolint#configure) for details. We can run `hadolint` quickly in the future with the configuration for `Dockerfile` scanning.

Reply:
Got it, thanks!